lib/takeaway.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'menu'
+require_relative 'twiliosms'
+
+class Takeaway
```

Comment:
Great application of SRP in your class methods, they literally do one single thing each